src/main/java/org/broadinstitute/hellbender/engine/spark/GATKSparkTool.java

```diff
@@ -62,11 +62,6 @@
     @ArgumentCollection
     protected IntervalArgumentCollection intervalArgumentCollection = requiresIntervals() ? new RequiredIntervalArgumentCollection() : new OptionalIntervalArgumentCollection();
 
-    @Argument(doc = "maximum number of bytes to read from a file into each partition of reads. " +
-            "Setting this higher will result in fewer partitions. Note that this will not be equal to the size of the partition in memory",
-            fullName = "bamPartitionSize", shortName = "bps", optional = true)
-    protected long bamPartitionSplitSize = ReadsSparkSource.DEFAULT_SPLIT_SIZE;
-
```

Comment:
It would also be good if there were a mechanism (eg., an overridable method) in the `GATKSparkTool` interface to allow the default to be permanently overridden on a per-tool basis.